### PR TITLE
Docs-clarify-ippools

### PR DESCRIPTION
**Pull request by bmckercher123**
## Address user feedback on IP pools

https://tigera.atlassian.net/browse/DOCS-151
https://tigera.atlassian.net/browse/DOCS-105

- Clarify default IP pools and fix parameter
- Simply IP pool examples
- Merge to nightly 
-  Please cherrypick to EE (I don't know how to pick from OS to EE 3.6)

### CODE REVIEW

File: networking/legacy-firewalls.md [hunk 1]
```diff
@@ -58,15 +58,15 @@ Also, cluster administrators must have [configured IP pools]({{ site.baseurl }}/
...
+`cni.projectcalico.org/ipv4pools: "["pool-1", "pool-2"]"`
 
 Note the use of the escaped \" for the inner double quotes around the pool names.
 
 #### Restrict all pods within a namespace to use an IP address range
 
  Annotate the namespace with key `cni.projectcalico.org/ipv4pools` and/or `cni.projectcalico.org/ipv6pools` and value set to a list of IP pool names, enclosed in brackets.  For example:
 
-`cni.projectcalico.org/ipv4pools: "[\"pool-1\", \"pool-2\"]"`
+`cni.projectcalico.org/ipv4pools: "["pool-1", "pool-2"]"`
```
Comment (caseydavenport) [inline networking/legacy-firewalls.md:69]: This change is not correct - we still need the backticks here, or we need to use different quotation marks.

e.g.,

```
cni.projectcalico.org/ipv4pools: '["pool-1", "pool-2"]'
```

Comment (caseydavenport) [reply networking/legacy-firewalls.md:69]: Same for the above

File: networking/change-block-size.md [hunk 2]
```diff
@@ -19,9 +19,67 @@ This how-to guide uses the following {{site.prodname}} features:
 
 ### Concepts
 
+#### About IP pools
+
+The {{site.prodname}} IPAM block sizes for IP pools are:
```
Comment (caseydavenport) [inline networking/change-block-size.md:24]: I think we should say something like this.

By default, Calico uses an IPAM block size of 64 addresses - /26 for IPv4, and /122 for IPv6. However, the block size can be changed depending on the IP pool address family.
- IPv4: 20-32, inclusive
- IPv6, 116-128, inclusive.

File: networking/change-block-size.md [hunk 3]
```diff
@@ -19,9 +19,67 @@ This how-to guide uses the following {{site.prodname}} features:
...
+   # Configures Calico networking.
+   calicoNetwork:
+     # Note: The ipPools section cannot be modified post-install.
+    ipPools:
+    - blockSize: 26
+      cidr: 10.48.0.0/21
+      encapsulation: IPIP
+      natOutgoing: Enabled
+      nodeSelector: all()
+    - blockSize: 116
```
Comment (caseydavenport) [inline networking/change-block-size.md:47]: The example actually has two IPv4 pools specified - I'd suggest changing the latter to this:

```
- blockSize: 122
  cidr: 2001::00/64
  encapsulation: None
  natOutgoing: Enabled
  nodeSelector: all()
```

File: networking/change-block-size.md [hunk 4]
```diff
@@ -19,9 +19,67 @@ This how-to guide uses the following {{site.prodname}} features:
...
+      encapsulation: IPIP
+      natOutgoing: Enabled
+      nodeSelector: all()
+
+```
+
 #### Expand or shrink IP pool block sizes
 
-By default, the {{site.prodname}} IPAM block size for an IP pool is /26. To expand from the default size /26, lower the `blockSize` (for example, /24). To shrink the `blockSize` from the default /26, raise the number (for example, /28). 
+- To expand a `blockSize`, lower the size. For example, if /26, lower to /24.
```
Comment (caseydavenport) [inline networking/change-block-size.md:81]: I think I prefer the previous wording - what is the new text trying to improve?

File: networking/change-block-size.md [hunk 5]
```diff
@@ -152,7 +210,7 @@ calicoctl delete ippool default-ipv4-ippool
 
 #### Create a new IP pool with the desired block size
 
-In this step, we update the IPPool with the new block size of (/28).
+In this step, we update the IPPool with the new block size of (/28) for IPv4.
```
Comment (caseydavenport) [inline networking/change-block-size.md:213]: I don't understand this change either - what was wrong with the previous?

File: networking/change-block-size.md (hunk 2 above)
Comment (bmckercher123) [reply networking/change-block-size.md:24]: Fixed.

File: networking/change-block-size.md (hunk 3 above)
Comment (bmckercher123) [reply networking/change-block-size.md:47]: Fixed.

File: networking/change-block-size.md (hunk 4 above)
Comment (bmckercher123) [reply networking/change-block-size.md:81]: Reverted

File: networking/change-block-size.md (hunk 5 above)
Comment (bmckercher123) [reply networking/change-block-size.md:213]: Reverted

File: networking/legacy-firewalls.md (hunk 1 above)
Comment (bmckercher123) [reply networking/legacy-firewalls.md:69]: Fixed.

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**